server/models/service-instance.js

```diff
@@ -7,9 +7,15 @@ module.exports = function extendServiceInstance(ServiceInstance) {
     function(ctx, _, next) {
       debug('updateAttributes with %j', ctx.args.data);
       // When updating the instance via REST, only allow changes to cpus
-      ctx.args.data = {
-        cpus: ctx.args.data.cpus
-      };
+      // and enableTracing
+      var reqData = ctx.args.data;
+      ctx.args.data = {};
+      if (reqData.cpus)
```

Comment:
Yah, I need to set the fields only if they are not null otherwise the update might fail